tests/e2e/specs/cli/index.test.js

```diff
@@ -111,24 +92,28 @@ describe("CLI", () => {
 
   test(
     "should successfully build a pdf",
-    async done => {
+    async (done) => {
       const testFilePath = path.join("static", "downloads", "test");
       await quire.pdf(testFilePath);
-      assert.equal(
-        fs.existsSync(`${testFilePath}.pdf`),
-        true
-      );
+      assert.equal(fs.existsSync(`${testFilePath}.pdf`), true);
       done();
     },
     timeout
   );
 
   test(
     "quire imageslice should create 'img/iiif/processed' image directory",
-    async done => {
-      await quire.process('iiif');
+    async (done) => {
+      await quire.process("iiif");
       assert.equal(
-        fs.existsSync(path.join(sandboxDir.name, projectName, "static", "img", "iiif", "processed")),
+        fs.existsSync(
+          path.join(
+            "static",
+            "img",
+            "iiif",
+            "processed"
+          )
+        ),
```

Comment:
This could be written more clearly as,
```
  test(
     "quire imageslice should create 'img/iiif/processed' image directory",
     async (done) => {
      await quire.process("iiif");
      const outputPath = path.join("static", "img", "iiif", "processed");
      assert(fs.existsSync(outputPath).toBe(true);
      done();
     },
     timeout
   );
```